### ISSUE TITLE
Compute s_region

### ISSUE THREAD

**Issue by stscijgbot-rstdms** [spacetelescope/romancal#418]
_Issue [RCAL-295](https://jira.stsci.edu/browse/RCAL-295) was created on JIRA by [Nadia Dencheva](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dencheva):_

The wcsinfo schema defines "s_region" already but it's not computed by the pipeline.

Needed by CAOM.

At this time we can populate it for L2 products. In order to populate it for L1 files we need to develop "set_telescope_pointing". As a minimum we need access to the SIAF where the vertices of each aperture in the Ideal Frame are recorded. Also this assumes the same pointing transforms as JWST (i.e. the same coordinate frames associated with the telescope and used by the SIAF) though we have no confirmation on that.

[Megan Sosey](https://jira.stsci.edu/secure/ViewProfile.jspa?name=sosey) Is a value of s_region for L2 products only sufficient at this time?

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Megan Sosey](https://jira.stsci.edu/secure/ViewProfile.jspa?name=sosey) on [JIRA](https://jira.stsci.edu/browse/RCAL-295?focusedCommentId=577180#comment-577180):_

[Nadia Dencheva](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dencheva) value of s_region for L2 is good for now.

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Nadia Dencheva](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dencheva) on [JIRA](https://jira.stsci.edu/browse/RCAL-295?focusedCommentId=591729#comment-591729):_

S_REGION is defined in the STC VO standard. It is used by the archive and is essentially a FITS string keyword. We are adding a string attribute in the same format to the asdf metadata, under wcsinfo. It needs to define the footprint of an observation on the sky. There is a method in gwcs which computed the footprint.

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Mairan Teodoro](https://jira.stsci.edu/secure/ViewProfile.jspa?name=mteodoro) on [JIRA](https://jira.stsci.edu/browse/RCAL-295?focusedCommentId=592852#comment-592852):_

[Nadia Dencheva](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dencheva)

Added the footprint (S_REGION) to the metadata, under wcsinfo, as following:


```java
{
   'v2_ref': 1312.9491452484797,
   'v3_ref': -1040.7853726755036,
   'vparity': -1,
   'v3yangle': -60,
   'ra_ref': 84.49289366006334,
   'dec_ref': -69.14101326380924,
   'roll_ref': 0.0,
   's_region': 'POLYGON IRCS 84.72986982015597 -69.1172110260591 84.43063054296834 -69.0558095642187 84.25268350051954 -69.16447822363403 84.55359892597048 -69.22555738863907 '
}```

**Comment by nden** [comment]
Fixed in #539
